vendor/bat-native-ledger/src/bat/ledger/ledger.cc

```diff
@@ -63,7 +63,7 @@ bool VisitData::loadFromJson(const std::string& json) {
 
   // has parser errors or wrong types
   bool error = d.HasParseError();
-  if (false == error) {
+  if (!error) {
```

Comment:
nit: code would be easier to read as follows:
```
bool success = !d.HasParseError();
if (!success) {
  // Fail...
}
```

Reply:
agree, but wanted to minimise changes